torch/_inductor/scheduler.py

```diff
@@ -1901,9 +1856,6 @@ def replace_buffer(
                 new_scheduler_node.min_order = node.min_order
                 new_scheduler_node.max_order = node.max_order
                 new_scheduler_node.last_usage = node.last_usage
-                for user in new_scheduler_node.users:
-                    user.node.inverse_users.remove(node)
-                    user.node.inverse_users.append(new_scheduler_node)
```

Comment:
where is this done now? Is it recomputed every time is needed?

Reply:
The comms passes are only called once so this was pointless book-keeping. We compute it once and discard it when the pass is finished, so don't need to keep it up to date.